gucci_test.go

```diff
@@ -24,6 +24,13 @@ func TestFuncShell(t *testing.T) {
 	}
 }
 
+func TestFuncShellArguments(t *testing.T) {
```

Comment:
Can we add a test for the scenario of using `.guest` in the shell command

Reply:
I guess you're referring to the example in the README

```
# guest: world
{{ shell "echo hello " .guest }}
```

I didn't add any specific test for that, since the substitution is happening "outside the `shell` function. I mean, `shell` will be receiving "echo hello" and "world" strings.

Does it make sense?
Thank you